docs/src/stake-pool/fees.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Fees
+---
+
+The stake pool program provides managers many options for making the pool
+financially viable, predominantly through fees. There are five different sources
+of fees.
```

Comment:
I would suggest adding a small disclaimer here that operators of the pool should take time to understand the purpose of the fees  and think about them carefully to ensure that the pool cannot be abused. Right now this looks like a reference that some might just skim quickly.

Reply:
That's a good point, Will do